Makefile

```diff
@@ -44,3 +44,11 @@ endif
 	git -C site commit -m '$(GITHUB_REF:refs/tags/v%=%)' index.html
 	git -C site push
 .PHONY: site-publish
+
+
+manpage:
+	mkdir -p $@
+
+.PHONY: manpages
+manpages: manpage
+	go run ./cmd/gen-docs --man-page --doc-path ./manpage
```

Comment:
Would we consider generating man pages under `share/man/man1/` in the project directory and add them like that to release tarballs? This would be so that the path of the files in the tarball rougly map to their placement under PREFIX when installed. We already do that with the binary: `bin/gh`

Reply:
I like that idea for the output, but I think having a directory called `share` in the project root might be confusing?
If there was a build outputs dir this wouldn't be a problem.

Reply:
fwiw I've seen other commands end up with a `share/` in their project root :woman_shrugging: I think it's ok.